forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -261,9 +296,9 @@ pub struct Command {
     #[clap(long, default_value = "default", help_heading = "OUTPUT")]
     pub output: OutputFormat,
 
-    /// Show transaction receipts in the output
-    #[clap(long, short = 'r', alias = "receipts", help_heading = "OUTPUT")]
-    pub show_receipts: bool,
+    /// Set verbosity levels; currently only supports max 2 levels
```

Comment:
can we leave a brief description about what each level of verbosity outputs here.

Reply:
https://github.com/FuelLabs/sway/pull/7074/commits/65383701488f2e89999779c5238f7c959e12e071